projects/frontend/data-pipelines/gui/projects/ui/src/assets/data/appConfig.json

```diff
@@ -36,5 +36,7 @@
       "remainingTime": 360,
       "checkInterval": 60
     }
-  }
+  },
+  "ignoreComponents": [],
+  "ignoreRoutes": []
```

Comment:
Do we include any default configurations on hidden pages/routes, for example Explore Data Jobs feature?
I don't see any documented examples on potential configurations of these new options provided?

Reply:
I'm not sure what the suggestion is here. We have examples as part of the tests. These will also be included in the helm chart config.

Reply:
Can you please link to the example on how to configure those settings, for hiding the Explore Data Jobs feature?